### ISSUE TITLE
Tooltips on Opcodes and Registers

### ISSUE THREAD

**Issue by rickmark** [Vector35/binaryninja-api#3864]
**What is the feature you'd like to have?**
Hovering over an opcode or register in disassembly showing documentation provided by a plugin.

This would make it easier to lookup unusual opcodes or system registers.

Implementation could be for the plugins to be able to register a provider for documentation, probably in RST or Markdown, that would be displayed.

**Is your feature request related to a problem?**
Quickly understanding an opcode

**Are any alternative solutions acceptable?**
Today it requires going outside of binja to the web

**Additional Information:**
Additional MSRs should also be able to be documented

**Comment by galenbwill** [comment]
This does not address the full scope of your request, but have you checked out this excellent plugin? https://github.com/ehennenfent/binja_explain_instruction

**Comment by rickmark** [comment]
Yep - the extra bits there are the registers and the tooltip UI idiom, just wanted to capture as a nice to have
